### ISSUE TITLE
Simple stdout/syslog configuration gives scheduler errors during run

### ISSUE THREAD

**Issue by xaque208** [fluent/fluent-bit#1613]
## Bug Report

**Describe the bug**
Using stdout output and syslog input, I'm not able to see logs that should be showing up on the UDP listener.

**To Reproduce**
Simply running `fluent-bit -v --config /usr/local/etc/fluent-bit/fluent-bit.conf` in my environment, with a nearly default config as follows.

```
[SERVICE]
    Flush        5
    Daemon       Off
    Log_Level    info
    Parsers_File parsers.conf
    Plugins_File plugins.conf
    HTTP_Server  Off
    HTTP_Listen  0.0.0.0
    HTTP_Port    2020
    Coro_Stack_Size 24576

[INPUT]
    Name     syslog
    Mode     udp
    Listen   0.0.0.0
    Port     5140

[OUTPUT]
    Name   stdout
    Match  *
```

```
$ fluent-bit -v --config /usr/local/etc/fluent-bit/fluent-bit.conf
Fluent Bit v1.0.4
Copyright (C) Treasure Data

[2019/10/04 13:42:47] [ info] [storage] initializing...
[2019/10/04 13:42:47] [ info] [storage] in-memory
[2019/10/04 13:42:47] [ info] [storage] normal synchronization mode, checksum disabled
[2019/10/04 13:42:47] [ info] [engine] started (pid=15929)
[2019/10/04 13:42:47] [ info] [in_syslog] UDP buffer size set to 32768 bytes
[2019/10/04 13:42:47] [ info] [in_syslog] UDP server binding 0.0.0.0:5140
[2019/10/04 13:42:57] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:07] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:17] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:27] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:37] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:47] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:43:57] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
[2019/10/04 13:44:07] [error] [src/flb_scheduler.c:47 errno=0] No error: 0
```

**Expected behavior**
First time trying to use fluent-bit, but I think its supposed to output something when a log message comes in.

**Your Environment**

* Version used: v1.0.4
* Configuration: See above
* Environment name and version (e.g. Kubernetes? What version?): ?
* Server type and version: Jailed, but shouldn't matter.
* Operating System and version:  FreeBSD 12.0-RELEASE
* Filters and plugins:

**Additional context**

I'm just trying to see if fluent bit will be a good fit for my logging deployment using.  Testing an integration with New Relic.

**Comment by edsiper** [comment]
can you try the latest v1.3.1 version ?

**Comment by xaque208** [comment]
Same issue, different line number for the error:

```
Starting fluent_bit.
[2019/10/09 18:14:22] [ info] [storage] initializing...
[2019/10/09 18:14:22] [ info] [storage] in-memory
[2019/10/09 18:14:22] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2019/10/09 18:14:22] [ info] [engine] started (pid=25184)
[2019/10/09 18:14:22] [ info] [in_syslog] UDP buffer size set to 32768 bytes
[2019/10/09 18:14:22] [ info] [in_syslog] UDP server binding 0.0.0.0:5140
[2019/10/09 18:14:22] [ info] [sp] stream processor started
[2019/10/09 18:14:32] [error] [src/flb_scheduler.c:48 errno=0] No error: 0
[2019/10/09 18:14:42] [error] [src/flb_scheduler.c:48 errno=0] No error: 0
[2019/10/09 18:14:52] [error] [src/flb_scheduler.c:48 errno=0] No error: 0
```

Running the version command without the config specification:
```
[root@sv2 ~]$ fluent-bit version
Fluent Bit v1.3.1
Copyright (C) Treasure Data

Error: Invalid coroutine stack size. Aborting
```

**Comment by edsiper** [comment]
if you remove ```Coro_Stack_Size``` from your config, do you see the same issue ?

**Comment by xaque208** [comment]
Without that set, I get the following.

```
$ /usr/local/bin/fluent-bit --config /usr/local/etc/fluent-bit/fluent-bit.conf --verboseFluent Bit v1.3.2
Copyright (C) Treasure Data

Error: Invalid coroutine stack size. Aborting
```

**Comment by dellelce** [comment]
@edsiper I have the same issue while testing  an almost default build of 1.5.0

**Comment by edsiper** [comment]
@dellelce did you build it from scratch ? if so, which cmake options ?

**Comment by dellelce** [comment]
Yes,built from scratch:

https://github.com/dellelce/mkit/blob/master/modules/fluentbit/hooks/configure.sh

```
cmake "${srcdir_fluentbit}"  -DCMAKE_INSTALL_PREFIX=${prefix} \
                              -DFLB_RECORD_ACCESSOR=Off \
                              -DFLB_STREAM_PROCESSOR=Off
```

**Comment by girgen** [comment]
Hi, this is indeed a problem still. I'm maintaining the FreeBSD port, and I now get 
```
Error: Invalid coroutine stack size. Aborting
``` as above. 
Here's a discussion about this. How can this best be fixed?

**Comment by girgen** [comment]
I believe this was fixed in #3716
